coupling/paralleltopology/XYZTopology.h

```diff
@@ -66,7 +66,7 @@ template <unsigned int dim> class coupling::paralleltopology::XYZTopology : publ
       index = _numberProcesses[d] * index + processCoordinates[d];
     }
 #if (COUPLING_MD_DEBUG == COUPLING_MD_YES)
-    std::cout << "Process coordinates=" << processCoordinates << " correspond to rank=" << index + _topologyOffset << std::endl;
+    // std::cout << "Process coordinates=" << processCoordinates << " correspond to rank=" << index + _topologyOffset << std::endl;
```

Comment:
Why is this commented out and not removed? This code is only compiled if debugging is enabled anyway.

Reply:
In case of ENABLE_COVERAGE=ON then we are in Debug mode, I would suggest to not have these flags in that case. But in ENABLE_COVERAGE=OFF and Debug mode, I would keep the print out.

Reply:
@rubenhorn because we still need this code for debugging in some cases
@LouieVoit Great idea! Thanks. I did that now (6ce0fc22c043)